lms/static/sass/course/instructor/_cohort-management.scss

```diff
@@ -0,0 +1,143 @@
+.cohort-management {
+
+  .has-option-unavailable { // Given to <selects> that have some options that are unavailable
+
+    .option-unavailable { // Allows granular styling (i.e., red) for those unavailable options
+      background: $red-l5;
+      color: $error-red;
+    }
+  }
+
+  .has-other { // Given to groups which have an 'other' input that appears when needed
+    display: inline-block;
+
+    label {
+      display: inline-block;
+    }
+
+    .input-group-other {
+      display: none;
+
+      &.is-visible {
+        display: inline;
+        position: relative;
+        overflow: auto;
+        width: 100%;
+        height: auto;
+        margin: 0 0 0 $baseline;
+        padding: inherit;
+        border: inherit;
+        clip: auto;
+      }
+    }
+  }
+
+  .cohort-management-nav {
+    margin-bottom: $baseline;
+
+    .action-create {
+      margin-top: $baseline;
+    }
+  }
+
+  .cohort-management-settings {
+    @include clearfix();
+    background: $gray-l5;
+
+    .cohort-manage-group-header {
+      padding: 0;
+      border-bottom: 0;
+
+      .group-header-title {
+        padding: $baseline;
+        margin-bottom: 0;
+        border-bottom: 1px solid $gray-l3;
+        text-transform: none;
+        letter-spacing: normal;
+      }
+
+      .cohort-management-group-setup {
+        padding: $baseline;
+
+        .cohort-management-group-text {
+          display: inline-block;
+          width: flex-grid(9);
+        }
+
+        .cohort-management-group-actions {
+          display: inline-block;
+          width: flex-grid(3);
+          vertical-align: top;
+
+          .float-right {
```

Comment:
I thought I made a previous comment about this rule. Why can't this property be placed on a specific UI element's class name rather than be a utility class. This seems like a bad idea to 1) creating utility styling for basic floating at this point, and 2) add into this layer of our Sass (scoped to a particular view). Please remove and refactor.